bin/main

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+puts 'Hello world!'
```

Comment:
* [ ] Your file should be executable using `bin/main.rb` as well as `ruby bin/main.rb` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Right now, typing `bin/main.rb` returns a `permission denied` error.
Check this [link](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) and make sure you followed all steps.

Also, I would advise you against removing the `.rb` extension of your file. It's not a good practice.

Reply:
Hi majovanilla, 
Thank you for your review, I have renamed the files, the configuration for permissions are done on the local machine thus for that to run you will have to configure on your local machine.